lingpy/convert/strings.py

```diff
@@ -404,40 +401,69 @@ def multistate2nex(taxa, matrix, filename='', missing="?"):
     return
 
 
+def nexus_slug(s):
```

Comment:
sorry for bothering on this: but I'd still say: the ideal way is to just use:

```python
slug_languagename = lambda s: clldutils.misc.slug(s, remove_whitespace=False, lowercase=False).replace(' ', '_')
```

Reply:
I mean, not literally, but similarly, lambda of course not needed ;-)

Reply:
Besser?